src/estrutura_sequencial/exercicio_tres.py

```diff
@@ -0,0 +1,13 @@
+"""
+Faça um Programa que peça dois números e imprima a soma.
+"""
+
+def imprime_resultado(soma):
+    return f"A soma de N1 + N2 e de {soma}"
+
+def main():
+    N1 = input("Digite aqui o primeiro número: ")
+    N2 = input("Digite aqui o segundo número: ")
+    soma = N1 + N2
+    mensagem = imprime_resultado(soma)
+    print(mensagem)
```

Comment:
aqui tá muito acoplado o algoritmo de execução ao de soma, recomendo desacoplar a função de soma.

```suggestion

def soma(num_um, num_dois):
    return num_um + num_dois
    
def main():
    num_um = input("Digite aqui o primeiro número: ")
    num_dois = input("Digite aqui o segundo número: ")
    resultado = soma(num_um, num_dois)
    mensagem = imprime_resultado(resultado)
    print(mensagem)
```